lib/coinbase/webhook.rb

```diff
@@ -0,0 +1,167 @@
+# frozen_string_literal: true
+
+module Coinbase
+  # A representation of a Webhook.
+  # This class provides methods to create, list, update, and delete webhooks
+  # that are used to receive notifications of specific events.
+  class Webhook
+    class << self
+      # Creates a new webhook for a specified network.
+      #
+      # @param network_id [String] The network ID for which the webhook is created.
+      # @param notification_uri [String] The URI where notifications should be sent.
+      # @param event_type [String] The type of event for the webhook. Must be one of the following:
+      #   - `WebhookEventType::UNSPECIFIED` ("unspecified")
+      #   - `WebhookEventType::ERC20_TRANSFER` ("erc20_transfer")
+      #   - `WebhookEventType::ERC721_TRANSFER` ("erc721_transfer")
```

Comment:
Can we add constants on this class e.g. `Coinbase::Webhook::ERC20_TRANSFER_EVENT`?